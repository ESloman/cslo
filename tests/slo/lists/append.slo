var y = [];
print(y);

y.append(1);
print(y);

for (var x = 1; x < 10; x++) {
    var ran = x**x;
    y.append(ran);
}
print(y);
