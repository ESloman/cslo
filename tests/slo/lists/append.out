list[0]: []
list[1]: [1]
list[10]: [1, 1, 4, 27, 256, 3125, 46656, 823543, 1.67772e+07, 3.8742e+08]
